From ca0d1296f2b9d63a1a80f07e85fe2e97df981084 Mon Sep 17 00:00:00 2001
From: Zhou weixin <zwx@rock-chips.com>
Date: Fri, 20 Jan 2017 10:07:53 +0800
Subject: [PATCH] ARM64: dts: rk3399-tve1205g: add some config for uboot charge

1. add charge ic support max input voltage and current for fusb302 pd.
2. set uboot-exit-charge-level to 2.

Change-Id: I41558bd3d72c4ad8cd4392c6cbedb4b1ebf6b28c
Signed-off-by: Zhou weixin <zwx@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts b/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
index b551d0bceec4..565fb76f7506 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
@@ -288,9 +288,11 @@
 		compatible = "rockchip,uboot-charge";
 		rockchip,uboot-charge-on = <1>;
 		rockchip,android-charge-on = <0>;
-		rockchip,uboot-exit-charge-level = <3>;
+		rockchip,uboot-exit-charge-level = <2>;
 		rockchip,uboot-low-power-level = <1>;
 		rockchip,uboot-charge-brightness = <0>;
+		max-input-voltage = <20000>;
+		max-input-current = <6000>;
 	};
 
 	vibrator {
-- 
2.35.3

